CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# This is the main CMake file for NCEPLIBS-wrf_io.
+#
+# Rahul Mahajan
```

Comment:
Please remove names. 
There is no I. 
git-blame should be enough to identify the contributions.

Reply:
Strongly disagree.

Reply:
Well, please remove my name.

Reply:
As is customary with science projects we will allow each contributor to add their name to the finished product. 

One purpose of the list of authors in a function is that it is the list of people who take responsibility for that function.

Tracking authorship has benefit besides fairness and historical record. Programmers crave legacy and too often this craving is satisfied by the creation of unneeded code. This adds to project complexity and costs. 

Instead, we will allow everyone to take credit for all the needed code they have contributed, and remove unneeded code. In this way, programmers are encouraged to build legacy as they should - by make valid contributions to necessary code.

Given all this, do you still want me to remove your name?

Reply:
I take ownership of the code I write. And that is why I am responding to every issue and PR that was my work. 
I do not crave for credit. And I despise legacy where authors like to add their names for every single line of contributions. 
Yes, I wish to remove my name.

Reply:
As you wish.